build.py

```diff
@@ -44,8 +44,8 @@
   'out' : 'bin',
   'src' : [ 'src', 'test' ],
   'libraries' : [
-    'third_party/junit4-4.11.jar',
-    'third_party/hamcrest-core-1.3.jar'
+    'third_party\junit4-4.11.jar',
```

Comment:
You shouldn't commit the changes to this file, since they are specific to windows. Are these changes actually necessary in order to build on windows? Either way, you should undo these changes.